modules/plugin/imagemosaic/pom.xml

```diff
@@ -207,20 +207,4 @@
   </dependencies>
 
 
-  <build>
```

Comment:
Maybe someone on Windows can try the build and see if removing this causes issues? I've checked the history, but it has been there longer than what we have in git